man/publish_addin.Rd

```diff
@@ -0,0 +1,11 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/rstudio.R
+\name{publish_addin}
+\alias{publish_addin}
+\title{Add in to Publish an R Markdown file to Platform Reports from RStudio}
+\usage{
+publish_addin()
+}
+\description{
+Add in to Publish an R Markdown file to Platform Reports from RStudio
+}
```

Comment:
I actually think it's a bug that this is generating docs since it's not exposed.  (Although it is "exposed" in the sense that it's a button... so maybe it needs docs anyway).  Why don't we skip adding this now and fix it in another PR?

[here](https://github.com/civisanalytics/civis-r/blob/master/R/rstudio.R#L1) I believe the `#'` should be `#`

Reply:
Thanks for the PR!